src/index.js

```diff
@@ -12,6 +12,19 @@ function installRouterPrefetch(Vue, { componentName = 'RouterLink' } = {}) {
       })
     })
 
+    const requestIdleCallback = window.requestIdleCallback ||
```

Comment:
use [inBrowser](https://github.com/egoist/vue-router-prefetch/blob/master/src/utils.js#L1) to ensure it's in browser

Reply:
Done